live-hls.js

```diff
@@ -452,20 +455,23 @@ const ui = function(request, response) {
 
     for (key in streams) {
       if (key.indexOf('.m3u8') > -1) {
-        let streamPath;
+        let errorPath;
 
         if (key.indexOf('http') > -1) {
           // this is a remote stream
-          streamPath = `../error/${key}`;
+          errorPath = `../error/${key}`;
         } else {
           // local stream
-          streamPath = `../${key.replace('live', 'error')}`;
+          errorPath = `../${key.replace('live', 'error')}`;
         }
 
-        button = '<td><button onclick="injectError(\''+streamPath+'?errorcode=1\')">errortext</button></td>';
+        button = '<td><button onclick="makeRequest(\'url?errorcode\')">errortext</button></td>';
```

Comment:
I changed the name of the function in the HTML page for two reasons.  1 - it was a bit confusing given the injectError() function in this file too.  2 - the HTML page's function really just loads a URL as opposed to doing anything error-specific.